src/pages/Demo/Demo.js

```diff
@@ -25,6 +26,7 @@ const Demo = () => {
           <Button type='outlined' link='/donate'>Int Link (Donate)</Button>
         </div>
       </Container>
+      <CardDemo></CardDemo>
```

Comment:
This can be `<CardDemo />` since there are no children.

Reply:
@mcheng09 , fixed this in latest push